specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-07-02-preview/managedClusters.json

```diff
@@ -6158,6 +6162,21 @@
       },
       "description": "Workload Identity settings for the security profile."
     },
+    "ManagedClusterSecurityProfileImageCleaner": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable ImageCleaner on AKS cluster."
+        },
+        "intervalHours": {
+          "type": "integer",
+          "format": "int32",
+          "description": "ImageCleaner scanning interval."
+        }
+      },
+      "description": "ImageCleaner settings for the security profile."
```

Comment:
@miwithro - do we want to actually document/link to the underlying implementation here? Or do we have a documentation page we can link to that explains what this does?

IMO we need to say a bit more about what ImageCleaner is doing. Something like: `ImageCleaner removes unused images from nodes, freeing up disk space and helping to reduce attack surface area. For more information see [this article](link-does-not-exist)

Reply:
@matthchr @jiashun0011 I agree with adding more context in the API.  Worst case we can add a link to the document.

Reply:
@matthchr @miwithro 

Fixed and because we don't have any external docs to clients, so I didn't attach the doc link in the description.